internal/filtering/statefilter.go

```diff
@@ -33,10 +31,15 @@ func FilterState(state *terraform.State) (*terraform.State, error) {
 		if resource.Mode != "managed" {
 			continue
 		}
+
+		allowedAttributes := []string{"id"}
```

Comment:
Could be nice encapsulate that logic and return that directly from `GetAllowedAttributes`

Reply:
This is what I was suggesting here yeah 👍  https://github.com/snyk/snyk-iac-capture/pull/14/files/e0561c49ddf38c913eb3b49fa795d6cd4e9c8d4d#r1132607896

Reply:
Misunderstood this one, sorry guys, should be fine now in [8990fe0](https://github.com/snyk/snyk-iac-capture/pull/14/commits/8990fe01bbc9379f01a9e1ff7ef6a1ada67f64f6)